dz5/ProjectDjango/middleware.py

```diff
@@ -0,0 +1,41 @@
+import logging
+
+import faker
+from django.utils.deprecation import MiddlewareMixin
+from uuid import uuid4
+
+class LogMiddleware(MiddlewareMixin):
+
+    def process_view(self, request, view_func, view_args, view_kwargs):
+        logging.basicConfig(level=logging.INFO)
+        logging.info("process {}".format(view_func))
```

Comment:
in case when you use logging you shouldn't use format
use %s, like:
`logging.info("process %s", view_func)`